### PR TITLE
Don't add PR comments for Dependabot.

### PR DESCRIPTION
**Pull request by desrosj**
Dependabot will never open a Trac ticket, so it's pointless to ask for one.

Dependeabot is also only configured to watch GitHub Actions for updates. So changes being made by the bot do not require the use of Playground for verification and testing.

Trac ticket: https://core.trac.wordpress.org/ticket/62221.

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: .github/workflows/pull-request-comments.yml [hunk 1]
```diff
@@ -128,6 +128,11 @@ jobs:
             const fs = require( 'fs' );
             const issue_number = Number( fs.readFileSync( './NR' ) );
 
+            // Don't comment for Dependabot.
+            if ( context.actor == 'dependabot' ) {
+              return;
+            }
+
```
Comment (johnbillion) [inline .github/workflows/pull-request-comments.yml]: Can this instead be achieved with an `if` directive on the step? For example:

```yaml
if: ${{ github.actor != 'dependabot' }}
```

Comment (desrosj) [reply .github/workflows/pull-request-comments.yml]: Ah, yes. I think you're right. Updated to use that approach instead.

**Review by johnbillion — APPROVED** [review]
Let's give it a try

**Comment by desrosj** [comment]
Merged in https://core.trac.wordpress.org/changeset/59370.

**Comment by desrosj** [comment]
Seems that this did not work based on the latest PR opened by Dependabot: #7806.

**Comment by desrosj** [comment]
Looks like the [correct value](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#responding-to-events) to use is `dependabot[bot]`.

**Review by Sanel0101 — COMMENTED** [review]
@desrosj

**Pull request closed without merging**